### PR TITLE
Even more styling

### PR DESCRIPTION
**Pull request by Markvanderuit**
Styling for most of the views. There's still a lot to be done, but I'll save that for later.

### CODE REVIEW

**Comment by mennoo1996** [comment]
Great work!

File: elements/gotofail-drawer.html [hunk 1]
```diff
@@ -39,53 +45,83 @@
                 font-weight: bold;
                 vertical-align: middle;
                 line-height: 100%;
+                color: black;
+                opacity: 0.8;
             }
 
             .icon-style {
-                color: #303F9F;
+                color: #3F51B5;
```
Comment (martijnjanssen) [inline elements/gotofail-drawer.html:53]: All these colors could be extracted. We can use color variables!

File: elements/new-user-dialog.html [hunk 2]
```diff
@@ -19,55 +19,122 @@
...
+                flex-basis: 200px;
+                margin: 16px 0;
+            }
+
+            #toggles {
+                display: flex;
+                /*flex-flow: row wrap;*/
+                flex-direction: row;
+                flex-wrap: wrap;
+                justify-content: center;
```
Comment (martijnjanssen) [inline elements/new-user-dialog.html:98]: All these properties can be removed by using the polymer flex classes.

File: elements/shot-caller-display.html [hunk 3]
```diff
@@ -7,34 +7,55 @@
     <template>
         <style include="iron-flex"></style>
         <style>
+            :host {
+                display: block;
+            }
+
             paper-button {
```
Comment (martijnjanssen) [inline elements/shot-caller-display.html:14]: So there is no styling in this element?

File: elements/shot-caller-display.html [hunk 4]
```diff
@@ -7,34 +7,55 @@
...
+
             paper-card {
-                @apply(--layout-horizontal);
+                background-color: white;
+                margin: 24px;
+                width: auto;
+                height: auto;
+
+                --paper-card-header: {
+                    /*border-top: 6px solid #303F9F;*/
```
Comment (martijnjanssen) [inline elements/shot-caller-display.html:24]: Commented.

File: elements/shot-caller-display.html (hunk 3 above)
Comment (Markvanderuit) [inline elements/shot-caller-display.html:14]: Not touched yet. Next PR. I don't want to have to explain why my PR is too large, for the third time. xD

File: public/stylesheets/timeline.css [hunk 5]
```diff
@@ -17,10 +21,16 @@ paper-toolbar {
...
+    box-shadow: 0px 0px 15px rgba(0,0,0,0.2) !important;
+}
+
+.narrow-layout > #drawer {
+    margin-top: 60px;
 }
 
 #scrim {
     z-index: 4 !important;
+    background-color: transparent !important;
```
Comment (martijnjanssen) [inline public/stylesheets/timeline.css:33]: :scream:

File: elements/gotofail-drawer.html (hunk 1 above)
Comment (Markvanderuit) [inline elements/gotofail-drawer.html:53]: Will do in next PR.

**Comment by martijnjanssen** [comment]
Looks good!

**Pull request merged**